c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -42,7 +42,7 @@ using namespace android;
 constexpr uint32_t MIN_W = 176;
 constexpr uint32_t MIN_H = 144;
 constexpr c2_nsecs_t TIMEOUT_NS = MFX_SECOND_NS;
-constexpr uint64_t kMinInputBufferSize = 2 * WIDTH_1K * HEIGHT_1K;
+constexpr uint64_t kMinInputBufferSize = 1 * WIDTH_1K * HEIGHT_1K;
```

Comment:
Could you please check 8k video playback with this change? Does it work for 8K as well?

Thanks,

Reply:
8K is verified. It works